src/modules/modals/CreateColumn.vue

```diff
@@ -1,11 +1,10 @@
 <template>
-	<NcModal v-if="showModal" size="large" @close="actionCancel">
+	<NcDialog v-if="showModal"
+		:name="t('tables', 'Create column')"
+		size="large"
```

Comment:
Could we also pass the buttons to the actions slot? That should make them sticky at the bottom of the modal so they are not hidden behind scroll bars